CMakeLists.txt

```diff
@@ -21,10 +22,8 @@ else()
   set(CPU_ONLY TRUE)
 endif()
 
-if(CPU_ONLY)
-  project(MaCh3 VERSION ${MaCh3_VERSION} LANGUAGES CXX)
-else()
-  project(MaCh3 VERSION ${MaCh3_VERSION} LANGUAGES CXX CUDA)
+if(NOT CPU_ONLY)
+  enable_language(CUDA)
```

Comment:
Could you move it a few line above?
around
set(MaCh3_GPU_ENABLED TRUE)

Reply:
yep